bazel/repository_locations.bzl

```diff
@@ -71,8 +71,8 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/google/protobuf/archive/v3.5.0.tar.gz"],
     ),
     envoy_api = dict(
-        commit = "258bfddfddaad4d6a1da7b4476c98b060d57bcc1",
-        remote = "https://github.com/envoyproxy/data-plane-api",
+        commit = "7d60aec93412f65768bb98cec6a9efb39956f197", # DO NOT SUBMIT
```

Comment:
just commenting so that once you merge in https://github.com/envoyproxy/data-plane-api/pull/500 you see this again